### PR TITLE
v2v: add TC "Clear error msg when no vpx username"

### PR DESCRIPTION
**Pull request by vwu-vera**
Refer bz#2181235

And refer https://github.com/avocado-framework/avocado-vt/pull/3725/

### CODE REVIEW

**Comment by vwu-vera** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.negative_test.inodes.less_inodes.esx_70.rhev.rhv.it_vddk.vpx_uri: PASS (59.57 s)

**Comment by vwu-vera** [comment]
@xiaodwan Please help to review. Thanks.:)

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -639,7 +640,7 @@ def vm_check(status_error):
 
         v2v_params = {
             'hostname': remote_host, 'hypervisor': 'esx', 'main_vm': vm_name,
-            'vpx_dc': vpx_dc, 'esx_ip': esx_ip,
+            'vpx_dc': vpx_dc, 'esx_ip': esx_ip, 'vpx_no_username': vpx_no_username,
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:643]: utils_v2v.params_get function can read all params which are define in tp-libvirt cfg files. 
We don't need to pass them into v2v_params anymore.

File: v2v/tests/src/function_test_esx.py [hunk 2]
```diff
@@ -50,6 +50,7 @@ def run(test, params, env):
     vpx_passwd = params.get("vpx_password")
     esxi_host = esx_ip = params.get('esx_hostname')
     vpx_dc = params.get('vpx_dc')
+    vpx_no_username = params.get('vpx_no_username')
```
Comment (xiaodwan) [inline v2v/tests/src/function_test_esx.py:53]: If using params_get, this line can be deleted.

File: v2v/tests/src/function_test_esx.py (hunk 1 above)
Comment (vwu-vera) [reply v2v/tests/src/function_test_esx.py]: Thanks. Have modified accordingly. :)

**Review by xiaodwan — APPROVED** [review]

**Comment by xiaodwan** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/3725

**Comment by vwu-vera** [comment]
@xiaodwan please help to merge. Thanks.

**Pull request merged**